src/main/platforms-config.js

```diff
@@ -2,7 +2,7 @@
  * @file Platforms configuration.
  *
  * It shall be overridden by custom configuration:
- * @see {@link https://github.com/AdguardTeam/FiltersRegistry/blob/master/custom_platforms.js}
+ * @see {@link https://github.com/AdguardTeam/FiltersRegistry/blob/master/scripts/build/custom_platforms.js}
```

Comment:
Could you please revert it or explain why this change is necessary?

Reply:
Because the former link is dead. 
![image](https://github.com/user-attachments/assets/512ffb64-31ed-4c24-9714-907bc7ca3246)

Reply:
Right, I didn't notice that.